regtests/ww3_ufs1.1/info

```diff
@@ -0,0 +1,60 @@
+#############################################################################
+#                                                                           #
+# ww3_ufs1.1 Test script for WW-III, 1deg global structured.                #
+#                                                                           #
+# Grids:                                                                    #
+# MODEL: glo_1deg with 1 deg resolution from 85S to 85N.                    #
+# INPUT: glo_30m.The forcing including ice concentration, wind speed (u10)  #
```

Comment:
this is now 15mext 

Also, I made this file: /scratch2/NCEPDEV/climate/Jessica.Meixner/forinputww3ufstest/gfs.20210401/00/atmos/wind.2021040100.nc which is from gfs output and has hourly wind for 2021040100, could we use this instead of the wind that is blocked out by ice?